auditbeat/module/auditd/config.go

```diff
@@ -120,11 +124,19 @@ func (c Config) rules() []auditRule {
 }
 
 func (c *Config) loadRules() error {
+	var log *logp.Logger
+	if c.IgnoreErrors {
+		log = logp.NewLogger(moduleName)
+	}
+
 	var paths []string
 	for _, pattern := range c.RuleFiles {
 		absPattern, err := filepath.Abs(pattern)
 		if err != nil {
-			return fmt.Errorf("unable to get the absolute path for %s: %w", pattern, err)
+			if log == nil {
+				return fmt.Errorf("unable to get the absolute path for %s: %w", pattern, err)
+			}
+			log.Warnf("unable to get the absolute path for %s: %v", pattern, err)
```

Comment:
just out of curiousity, why `%v` here for the error? I was under the impression `%w` was intended for printing errors

Reply:
`%w` is only for use with https://pkg.go.dev/fmt#Errorf when wrapping errors.